docs/content/en/docs/reference/baremetal/bare-prereq.md

```diff
@@ -15,26 +15,22 @@ Set up an Administrative machine as described in [Install EKS Anywhere ]({{< rel
 
 ## Compute server requirements
 
-The minimum number of physical machines needed to run EKS Anywhere in a non-production mode is:
-
-* Control plane physical machines: 1
-* Worker physical machines: 1
+The minimum number of physical machines needed to run EKS Anywhere on bare metal is 1. To configure EKS Anywhere to run on a single server, set [controlPlaneConfiguration.count]({{< relref "../clusterspec/baremetal/#controlplaneconfigurationcount-required" >}}) to 1, and omit [workerNodeGroupConfigurations]({{< relref "../clusterspec/baremetal/#workernodegroupconfigurations" >}}) from your cluster configuration. 
```

Comment:
This isn't rendering for me in preview mode. Have you tried:

```
[controlPlaneConfiguration.count](../clusterspec/baremetal#controlplaneconfigurationcount-required)
```

Reply:
resolved offline